Clarifying Update Group Usage. (#3004)
MicrosoftDocs/microsoft-365-docs: microsoft-365/managed-desktop/service-description/updates.md

- **Fast**: Prioritizes speed over stability. Useful for detecting quality issues before they are offered to the Broad group. This group serves as a next layer of validation but is generally more stable than the Test and First groups. 
- **Broad**: Last group to have feature and quality updates available. This group contains the majority of users in the tenant, and therefore favors stability over speed in deployment. Testing of apps should be done here as the environment is most stable. 

> ### Moving Devices between Update Groups

Reviewer: ```suggestion
### Moving devices between update groups
```

> You may want some devices to receive updates last and others that you want to go first. To move these devices into the appropriate Update Group [submit an administrator support request](https://docs.microsoft.com/en-us/microsoft-365/managed-desktop/working-with-managed-desktop/admin-support?view=o365-worldwide) and we will move the devices for you. 

Reviewer: ```suggestion
You may want some devices to receive updates last and others that you want to go first. To move these devices into the appropriate update group, [submit an administrator support request](https://docs.microsoft.com/microsoft-365/managed-desktop/working-with-managed-desktop/admin-support?view=o365-worldwide) and we will move the devices for you. 
```

> [!NOTE]

> > If you need to move a user to a different update group, submit a support request. Do not move devices between update groups yourself, there are serious consequences if a device is moved incorrectly: the device may update unexpectedly and policies may conflict, changing the device configuration.

Author: Please Change the word "User" to "Device" in the first sentence of this Note

> > If you need to move a user to a different update group, submit a support request. Do not move devices between update groups yourself, there are serious consequences if a device is moved incorrectly: the device may update unexpectedly and policies may conflict, changing the device configuration.

Reviewer: ```suggestion
> If you need to move a device to a different update group, submit a support request. Do not move devices between update groups yourself. There are serious consequences if a device is moved incorrectly; the device may update unexpectedly and policies may conflict, changing the device configuration.
```

> For more information roles and responsibilities with these deployment groups, see [Microsoft Managed Desktop Roles and responsibilities](../intro/roles-and-responsibilities.md)

Reviewer: ```suggestion
For more information on roles and responsibilities within these deployment groups, see [Microsoft Managed Desktop Roles and responsibilities](../intro/roles-and-responsibilities.md)
```

> ### Using Microsoft Managed Desktop Update Groups 

Reviewer: ```suggestion
### Using Microsoft Managed Desktop update groups 
```

> There are parts of the service, like app deployment, which you manage where it may be necessary to target all managed devices. In these instances, it makes sense to use the Update Groups to reach those users with the understanding that you cannot add, remove, or change the membership of those groups. 

Reviewer: ```suggestion
There are parts of the service, like app deployment, which you manage where it may be necessary to target all managed devices. In these instances, it makes sense to use the update groups to reach those users with the understanding that you cannot add, remove, or change the membership of those groups. 
```

    <tr><td>Anti-virus definition</td><td colspan="4">Updated with each scan</td></tr>
    <tr><td>Microsoft 365 Apps for enterprise</td><td colspan="4"><a href="https://docs.microsoft.com/microsoft-365/managed-desktop/get-started/m365-apps#updates-to-microsoft-365-apps">Learn more</a></td></tr>
    <tr><td>Microsoft Edge</td><td colspan="4"><a href="https://docs.microsoft.com/microsoft-365/managed-desktop/get-started/edge-browser-app#updates-to-microsoft-edge">Learn more</a></td></tr>

>     <tr><td>Microsoft Teams</td><td colspan="4"><a href="https://docs.microsoft.com/en-us/microsoft-365/managed-desktop/get-started/teams#updates">Learn more</a></td></tr>

Reviewer: ```suggestion
    <tr><td>Microsoft Teams</td><td colspan="4"><a href="https://docs.microsoft.com/microsoft-365/managed-desktop/get-started/teams#updates">Learn more</a></td></tr>
```
